src/extensions/lens-extension.ts

```diff
@@ -113,6 +112,11 @@ export class LensExtension {
     }
   }
 
+  @action
+  activate() {
+    return this.onActivate();
+  }
```

Comment:
This is unnecessary, why not just call `onActivate`?

Reply:
That's protected, and I didn't want to change the API interface, I thought that could affect extensions. But I could just make it a public method if that's fine.

Reply:
You just did make change the API interface though...

Reply:
Also, it seems strange that `enable` no longer calls `onActivate`. Could you also change its name? Because I find it a tad confusing why something would change `_enabled` while not calling `onActivate`

Reply:
I mean making `onActive` from protected to public is a breaking change, since extensions need to implement that interface, right? But here we're adding a new method, which is a non-breaking change.

I'll try to figure out better names tomorrow. Basically we need two steps: 1. "load" and 2. "enable".

Reply:
You could also just check the extensions and see if anyone also marks their overloads as protected too.

Reply:
That seems risky.. we don't necessarily know about all the extensions, some might be private. Breaking change is still a breaking change, and adding a new method seems fine to me.

Reply:
I think name `enable` is fine for that, and `activate` too. I can change if you can come up with something better.